### PR TITLE
Add release 6.1 post

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

File: src/news/release-6.1.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  - release
+---
+
+Hello!
+
+## Changelog
+
+### Changed
+
+- Move cat to the bottom right again by [@DioEgizio](https://github.com/DioEgizio) in [#612](https://github.com/PrismLauncher/PrismLauncher/pull/612)85)
```
Comment (DioEgizio) [inline src/news/release-6.1.md:18]: not really again, before 6.0 was on top right

File: src/news/release-6.1.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Fix importing modpacks using ZIP files by [@flowln](https://github.com/flowln) in [#607](https://github.com/PrismLauncher/PrismLauncher/pull/607)
+- Fix possible crash in mod and modpack downloaders with images by [@flowln](https://github.com/flowln) in [#605](https://github.com/PrismLauncher/PrismLauncher/pull/605)
+- Fix wordings in modpack updater by [@Scrumplex](https://github.com/Scrumplex) in [#601](https://github.com/PrismLauncher/PrismLauncher/pull/601)
+- Fix warning about QMap by [@Edgars-Cirulis](https://github.com/Edgars-Cirulis) in [#625](https://github.com/PrismLauncher/PrismLauncher/pull/625)
+- [Linux] Fix crash when using Qt GTK2 style by [@flowln](https://github.com/flowln) in [#604](https://github.com/PrismLauncher/PrismLauncher/pull/604)
+- [Windows] Fix automated WinGet releases by [@Scrumplex](https://github.com/Scrumplex) in [#593](https://github.com/PrismLauncher/PrismLauncher/pull/593) [#649](https://github.com/PrismLauncher/PrismLauncher/pull/649)
+
+### Removed
+
+- [Windows] Remove unused DLLs from builds by [@DioEgizio](https://github.com/DioEgizio) in [#642](https://github.com/PrismLauncher/PrismLauncher/pull/642)
```
Comment (DioEgizio) [inline src/news/release-6.1.md:37]: huh that also applies to macos tbh

File: src/news/release-6.1.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Move cat to the bottom right again by [@DioEgizio](https://github.com/DioEgizio) in [#612](https://github.com/PrismLauncher/PrismLauncher/pull/612)85)
+- Only show scrollbars when needed by [@leo78913](https://github.com/leo78913) in [#630](https://github.com/PrismLauncher/PrismLauncher/pull/630)
+- [Linux] Update Nix packaging by [@Scrumplex](https://github.com/Scrumplex) in [#647](https://github.com/PrismLauncher/PrismLauncher/pull/647)
+- [Windows] Improve MSVC build flags by [@getchoo](https://github.com/getchoo) in [#631](https://github.com/PrismLauncher/PrismLauncher/pull/631)
+
+### Fixed
+
+- Avoid deadlock when there are a lot of resource packs in an instance  by [@flowln](https://github.com/flowln) in [#628](https://github.com/PrismLauncher/PrismLauncher/pull/628)
+- Correctly set component versions in Modrinth pack updating by [@flowln](https://github.com/flowln) in [#636](https://github.com/PrismLauncher/PrismLauncher/pull/636)
```
Comment (flowln) [inline src/news/release-6.1.md:26]:
```suggestion
- Correctly set component versions in Modrinth pack updating by [@flowln](https://github.com/flowln) and [@Scrumplex](https://github.com/Scrumplex) in [#636](https://github.com/PrismLauncher/PrismLauncher/pull/636)
```

Comment (Scrumplex) [reply src/news/release-6.1.md:26]: oh GitHub API does not return co-authors. good to know

File: src/news/release-6.1.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  - release
+---
+
+Hello!
+
+## Changelog
+
+### Changed
+
+- Move cat to the bottom right by [@DioEgizio](https://github.com/DioEgizio) in [#612](https://github.com/PrismLauncher/PrismLauncher/pull/612)85)
```
Comment (jopejoe1) [inline src/news/release-6.1.md:18]:
```suggestion
- Move cat to the bottom right by [@DioEgizio](https://github.com/DioEgizio) in [#612](https://github.com/PrismLauncher/PrismLauncher/pull/612)
```
Nice random number in changelog

File: src/news/release-6.1.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
+---
+title: Prism Launcher Release 6.1, now available
+description: More addictive than lemonade!
+date: 2022-12-19
+release_version: "6.1"
+minimum_macos_version: 10.15.0
+mac_signature: 76S99OXja/WJbqYQvV9BoEAlTV4khWeZWCjcXKFHnXkgkZD18yXcBXAdY/BK0dhQrX1IbHZu3JKhEVyjFwKmCA==
```
Comment (DioEgizio) [inline src/news/release-6.1.md:7]:
```suggestion
mac_signature: 76S99OXja/WJbqYQvV9BoEAlTV4khWeZWCjcXKFHnXkgkZD18yXcBXAdY/BK0dhQrX1IbHZu3JKhEVyjFwKmCA==
```

**Pull request merged**